deployments/helm-chart/README.md

```diff
@@ -178,7 +178,7 @@ Parameter | Description | Default
 `controller.healthStatusURI` | Sets the URI of health status location in the default server. Requires `controller.healthStatus`. | "/nginx-health"
 `controller.nginxStatus.enable` | Enable the NGINX stub_status, or the NGINX Plus API. | true
 `controller.nginxStatus.port` | Set the port where the NGINX stub_status or the NGINX Plus API is exposed. | 8080
-`controller.nginxStatus.allowCidrs` | Whitelist IPv4 IP/CIDR blocks to allow access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas. | 127.0.0.1
+`controller.nginxStatus.allowCidrs` | Allowlist IPv4 IP/CIDR blocks access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas. | 127.0.0.1
```

Comment:
```suggestion
`controller.nginxStatus.allowCidrs` | Add IPv4 IP/CIDR blocks to the allow list for NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas. | 127.0.0.1
```

Per Microsoft style guide (which we follow), "allow list" should be two words, used as a noun (https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/whitelist)